neurots/extract_input/input_parameters.py

```diff
@@ -20,13 +20,24 @@
 tmd_algos = ("tmd", "tmd_gradient", "tmd_apical")
 
 
+def _sort_neurite_types(neurite_types):
+    """Sort neurite types to comply with internal requirements.
+
+    For now, only apical should be first, so it is placed first, and other `neurite_types` can be
+    placed with respect to it.
+    """
```

Comment:
any restrictions for the order of axons / basals?

Reply:
ah I forgot to remove that, it was bothering people, so I sort it externally now